modules/rosa-hcp-sts-creating-a-cluster-external-auth-cluster-cli.adoc

```diff
@@ -4,7 +4,7 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="rosa-hcp-sts-creating-a-cluster-external-auth-cluster-cli_{context}"]
-= Creating a {hcp-title} cluster that uses external authentication providers
+= Creating a {hcp-title} cluster with direct authentication with an external OIDC identity provider
```

Comment:
Same comment as the other module, about considering "that uses" instead of the middle "with". 

And also noting that the names in the topic map could also be updated. I'm really not positive if they're used anymore on docs.rh, but still would be good to make sure they look right on the preview.